### PR TITLE
feat: Lists and List page

### PR DESCRIPTION
**Pull request by nazreen**
Lists: all Lists
List: single List

List is a list of promises

Next:
* ListEditor (to add new List)
* PromiseEditor to have list_ids input

### CODE REVIEW

File: src/views/List.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="container">
+    <template v-if="appStatus === 'loading'">
+      <LoadingSpinner />
+    </template>
+    <template v-else>
+      <h1>{{ list.title }}</h1>
+    <promises-table :promises="promises" exclude="[source_name, politician_name]" />
+
```
Comment (sujono91) [inline src/views/List.vue]: remove this empty line

File: src/views/List.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    </template>
+  </div>
+</template>
+
```
Comment (sujono91) [inline src/views/List.vue]: same here

File: src/views/Lists.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    } catch (e) {
+      console.error(e)
+    }
+  },
+  methods: {
+    async listListsHandler () {
+      this.appStatus = 'loading'
+
+      return await listLists()
+
```
Comment (sujono91) [inline src/views/Lists.vue]: remove this empty line

File: src/api/index.js [hunk 4]
```diff
@@ -49,6 +49,30 @@ const updatePromiseUpdate = data => updateSomething(`${PROMISE_UPDATES_PATH}${da
...
+    id: 'ytughjbsdf8t7yiu',
+    title: 'Broadband service in Malaysia',
+    promise_ids: [
+      'XjGLsRtJ9spk5VeBidDl',
+      'bGvbmCPqIcpWQIZAp92C'
+    ]
+  }
+]
+
+const listLists = () => mockLists
```
Comment (sujono91) [inline src/api/index.js]: i just thought about a better naming convention, e.g. `getMockList()` WDYT?

Comment (nazreen) [reply src/api/index.js]: the idea is, once the API endpoint is ready, is to then replace `mockLists` with `getSomething()` call. check the other examples.
the interface, i.e. api.listLists should not be changed. the component that calls it (Lists) should not need to change the method it is calling

File: src/views/List.vue (hunk 2 above)
Comment (nazreen) [reply src/views/List.vue]: ok

File: src/views/Lists.vue (hunk 3 above)
Comment (nazreen) [reply src/views/Lists.vue]: ok

File: src/api/index.js (hunk 4 above)
Comment (sujono91) [reply src/api/index.js]: oh, yeah, i just saw the other fully implementation:
`const listContributors = () => getSomething('/contributors/')`

sorry @nazreen 🙏

File: src/views/List.vue (hunk 2 above)
Comment (nazreen) [reply src/views/List.vue]: this one serves a purpose thought. <template> and <script> in Vue's single component files are separate sections completely.

**Pull request merged**